### PR TITLE
Support deleting shards from TSDB store

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by otoolep** [comment]
@jwilder

File: tsdb/store.go [hunk 1]
```diff
@@ -65,6 +65,30 @@ func (s *Store) CreateShard(database, retentionPolicy string, shardID uint64) er
...
 
+// DeleteShard removes a shard from disk.
+func (s *Store) DeleteShard(shardID uint64) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	// ensure shard exists
+	sh, ok := s.shards[shardID]
+	if !ok {
+		return ErrShardNotFound
```
Comment (jwilder) [inline tsdb/store.go]: This should just return nil.

**Comment by jwilder** [comment]
:+1: after returning nil.

**Comment by otoolep** [comment]
Thanks @jwilder

**Pull request merged**